src/policy/p2p_messages/principal_messages.js

```diff
@@ -22,10 +22,7 @@ class MsgPrincipal {
   }
 
   toJson() {
-    return {
-      'requestMessage': this.getRequest(),
-      'workerSig': this.getSig(),
-    };
+    return [this.getRequest(),this.getSig()];
```

Comment:
Can you please explain this change?
Because I see that this has been changed (https://github.com/enigmampc/enigma-core/commit/ab41ca6124e279a4fe3bfb0d5da6de9bab364f06#diff-634bffb7870c18f8a700b8e03a1c4eb6) but to `data` and `sig`, not to an array.
what am I missing?

Reply:
This change went in here inadvertently. The proper change was included minutes ago in #134, but now has been merged to this PR for convenience in 1fb4fbe

(for the record, it works because if you pass it an array in the same order as the object properties, it will be processed in the same way, so it was a temporary hack to make it work regardless of the actual object properties in the jsonrpc server, but it's clearer to have it as an object as included in the commit referenced above.